condacolab.py

```diff
@@ -125,18 +152,16 @@ def install_from_url(
     if sitepackages not in sys.path:
         sys.path.insert(0, sitepackages)
 
-    print("🩹 Patching environment...")
     env = env or {}
     bin_path = f"{prefix}/bin"
-    if bin_path not in os.environ.get("PATH", "").split(":"):
-        env["PATH"] = f"{bin_path}:{os.environ.get('PATH', '')}"
-    env["LD_LIBRARY_PATH"] = f"{prefix}/lib:{os.environ.get('LD_LIBRARY_PATH', '')}"
 
     os.rename(sys.executable, f"{sys.executable}.real")
```

Comment:
I think we can be more informative now.

```suggestion
    os.rename(sys.executable, f"{sys.executable}.renamed_by_condacolab.bak")
```